<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QDialog" name="History">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>257</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <widget class="QComboBox" name="indexCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seller</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buyer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Animal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="paramaterLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>110</y>
     <width>180</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>History Param:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3DateEdit" name="startDateEntry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3DateEdit" name="endDateEntry">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>161</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reportButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Ending</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Index By:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Seller</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buyer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textLabel3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Starting:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="_historyParamaterEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>450</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h1&gt;History&lt;/h1&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3DateEdit</class>
   <extends>QWidget</extends>
   <header>Qt3Support/Q3DateEdit</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">types/historyitem.h</include>
  <include location="local">dao/historydao.h</include>
  <include location="local">dao/salemanager.h</include>
  <include location="local">dao/animalcodedao.h</include>
  <include location="local">autofilledit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
